app/controllers/schools/analysis_controller.rb

```diff
@@ -52,6 +52,10 @@ def storage_heaters
     render_generic_chart_template
   end
 
+  def carbon_emissions
```

Comment:
I wonder whether we should make this tab available for admin users only at the minute, else we will be sitting on this release without being able to push it to prod...